### ISSUE TITLE
[CoE Starter Kit - BUG] Mails preview in Command Center might be not aligned with flows - send via helper instead

### ISSUE THREAD

**Issue by lcimon** [microsoft/coe-starter-kit#2576]
### Describe the issue

Mails preview in Command Center are sent directly from the Power Apps. So it doesn't consider the customization done in 'HELPER - Send Email' if any.
As result, the preview sent by the app is different that the final email received by users.

### Expected Behavior

Preview emails should reflect the final email received.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.37

### What app or flow are you having the issue with?

Admin - Command Center

**Comment by Jenefer-Monroe** [comment]
Can you please give an example of what you mean here?

**Comment by lcimon** [comment]
I have customized the _HELPER - Send Email_ flow to include a banner when _ProductionEnvironment_ env var is set to false.
![image](https://user-images.githubusercontent.com/22662809/165535863-22c115c6-0fe0-4e9d-ac32-2b0cf33c1118.png)
My understanding is that change will impact all the mails sent by CoE.

Then, from Command Center, I updated the mail body of some emails and use the mail preview feature. Mails that I received don't have the banner that I added in the flow.

I understand that Command Center don't use the _HELPER - Send Email_ flow to send email preview but use the Outlook connector directly in Power Apps. This behavior might introduce a gap between the preview email and the final email really sent.

Hope that it's clearer.
